rootdir/vendor/ueventd.rc

```diff
@@ -149,24 +149,30 @@ firmware_directories /vendor/firmware_mnt/image/
-/sys/devices/platform/soc/c1b5000.i2c/i2c-7/7-0028  recv_rsp    0600 nfc nfc
-/sys/devices/platform/soc/c1b5000.i2c/i2c-7/7-0028  send_cmd    0200 nfc nfc
-
-# LED
-/sys/devices/platform/soc/800f000.qcom,spmi/spmi-0/spmi0-03/800f000.qcom,spmi:qcom,pm660l@3:qcom,leds@d000/leds/* max_brightness 0664 system system
-/sys/devices/platform/soc/800f000.qcom,spmi/spmi-0/spmi0-03/800f000.qcom,spmi:qcom,pm660l@3:qcom,leds@d000/leds/* brightness     0664 system system
-/sys/devices/platform/soc/800f000.qcom,spmi/spmi-0/spmi0-03/800f000.qcom,spmi:qcom,pm660l@3:qcom,leds@d000/leds/* breath          0664 system system
-/sys/devices/platform/soc/800f000.qcom,spmi/spmi-0/spmi0-03/800f000.qcom,spmi:qcom,pm660l@3:qcom,leds@d000/leds/* trigger      0664 system system
-/sys/devices/platform/soc/800f000.qcom,spmi/spmi-0/spmi0-03/800f000.qcom,spmi:qcom,pm660l@3:qcom,leds@d000/leds/* delay_off       0664 system system
-/sys/devices/platform/soc/800f000.qcom,spmi/spmi-0/spmi0-03/800f000.qcom,spmi:qcom,pm660l@3:qcom,leds@d000/leds/* delay_on       0664 system system
-/sys/devices/platform/soc/800f000.qcom,spmi/spmi-0/spmi0-03/800f000.qcom,spmi:qcom,pm660l@3:qcom,leds@d800/leds/wled  max_brightness 0664 system system
-/sys/devices/platform/soc/800f000.qcom,spmi/spmi-0/spmi0-03/800f000.qcom,spmi:qcom,pm660l@3:qcom,leds@d800/leds/wled  brightness     0664 system system
-/sys/class/backlight/panel0-backlight max_brightness                                                                                  0644 root system
-/sys/class/backlight/panel0-backlight brightness                                                                                      0664 system system
+/dev/pn553                                  0660 nfc nfc
+
+# LED class devices
+/sys/class/leds/red      max_brightness     0664 system    system
+/sys/class/leds/red      brightness         0664 system    system
+/sys/class/leds/red      breath             0664 system    system
+/sys/class/leds/red      trigger            0664 system    system
+/sys/class/leds/red      delay_off          0664 system    system
+/sys/class/leds/red      delay_on           0664 system    system
+/sys/class/leds/green    max_brightness     0664 system    system
+/sys/class/leds/green    brightness         0664 system    system
+/sys/class/leds/green    breath             0664 system    system
+/sys/class/leds/green    trigger            0664 system    system
+/sys/class/leds/green    delay_off          0664 system    system
+/sys/class/leds/green    delay_on           0664 system    system
+/sys/class/leds/blue     max_brightness     0664 system    system
+/sys/class/leds/blue     brightness         0664 system    system
+/sys/class/leds/blue     breath             0664 system    system
+/sys/class/leds/blue     trigger            0664 system    system
+/sys/class/leds/blue     delay_off          0664 system    system
+/sys/class/leds/blue     delay_on           0664 system    system
```

Comment:
Since ueventd support globs, how about `{red,green,blue}`?

Also, any idea why we were previously using the absolute path instead of these symlinks? The individual files were never symlinks, the surrounding folder was.

Reply:
I followed the style of the ueventd found on the EDO platform https://github.com/sonyxperiadev/device-sony-edo/blob/master/rootdir/vendor/ueventd.rc#L162, which was most likely done following the CAF variant
https://source.codeaurora.org/quic/la/device/qcom/common/tree/rootdir/etc/ueventd.qcom.rc?h=LA.UM.9.12.r1-12900-SMxx50.QSSI12.0#n381

I can update to `/sys/class/leds/{red,green,blue}`

Reply:
Just tried `/sys/class/leds/{red,green,blue}` and It didn't work. ueventd did not set permissions